src/components/ClusterSpinUp.js

```diff
@@ -1,17 +1,19 @@
 import React, {Component} from 'react';
 
 class ClusterSpinUp extends Component {  
-    clusterLinks = (url) =>   {
+    getUrlVars = (vars) => {
         window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
-        url[key] = value;
-    });
+        vars[key] = value;
+      });
     }
+
+    
     render() {
-        let url = { }
-        this.getUrlVars(url)
+        let vars = {}
+        this.getUrlVars(vars)
         return(
             <div>
-               <p>Url link</p>    
+              <iframe src = {vars.redirectUrl} title='Remote Desktop' height="1080" width="1920"/>
```

Comment:
Why are we using an iframe and not just redirecting the entire page? If we really want to use an iframe, the width and height have to be set dynamically based on clientWidth and clientHeight

Reply:
Because we still want to display a DWP 'banner' at the top of the page, together with a logoff button for instance.

Reply:
Note the width/height is set to the same height we have set the remote desktop to in the Orchestration service.

Reply:
Yep, we can add the dynamic size feature later when we also send the correct sizes to the orchestration service